deploy/crds/planetscale.com_vitessclusters.yaml

```diff
@@ -1989,6 +1993,184 @@ spec:
                   tolerations:
                     x-kubernetes-preserve-unknown-fields: true
                 type: object
+              vtAdmin:
```

Comment:
should be lowercase I think (same in the PR description)

Reply:
Yes, this makes sense. Fixed it now